Microsoft look back (#19066)
demisto/content: Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/README.md

    | First fetch time | First fetch timestamp \(&amp;lt;number&amp;gt; &amp;lt;time unit&amp;gt;, e.g., 12 hours, 7 days\) | False |
    | Incident resolution status |  | False |
    | Custom Filter | A custom filter by which to filter the returned files. If you pass the custom_filter argument it will override the other filters from the integration instance configuration. An example of a Custom Filter is: \{"severity":\{"eq":2\}\}. Note that for filtering by "entity.policy", you should use the ID of the policy. For example, for retrieving the policy: \{"policyType": "ANOMALY_DETECTION", "id": "1234", "label": "Impossible travel", "type": "policyRule"\}" please query on \{"entity.policy":\{"eq":1234\}\}. For more information about filter syntax, refer to https://docs.microsoft.com/en-us/cloud-app-security/api-alerts#filters. | False |

>     | lookback | Advanced: Minutes to look back when fetching. | False

Reviewer: ```suggestion
    | Advanced: Minutes to look back when fetching | Use this parameter to determine how long backward to look in the search for incidents to ensure collecting all incidents. | False
```
please update the yml too.
Also, what is the value we recommend?

5. Click **Test** to validate the URLs, token, and connection.

## look-back parameter note
In case the **look-back** parameter is initialized with a certain value and during a time that incidents were fetched, if changing 

> the look back to a number that is greater than the previous value, then in the initial incident fetching there will be incidents duplications.

Reviewer: ```suggestion
the look back to a number that is greater than the previous value, then in the next fetch there might be incidents duplications.
```

> If the integration was already set with look back > 0, and the look-back is not being increased at any point of time, then those incident duplications would not occur.

Reviewer: ```suggestion
```
